dataactvalidator/scripts/loadFile.py

```diff
@@ -284,20 +129,11 @@ def loadDomainValues(basePath, localSF133Dir = None, localProgramActivity = None
     else:
         loadProgramActivity(os.path.join(basePath, "program_activity.csv"))
 
-    # get a list of SF 133 files to load
-    sf133_list = getSF133List(localSF133Dir)
-    SF_RE = re.compile(r'sf_133_(?P<year>\d{4})_(?P<period>\d{2})\.csv')
-    for sf133 in sf133_list:
-        # for each SF file, parse out fiscal year and period
-        # and call the SF 133 loader
-        file_match = SF_RE.match(sf133.file)
-        if not file_match:
-            logger.info('{}Skipping SF 133 file with invalid name: {}'.format(
-                os.linesep, sf133.full_file))
-            continue
-        logger.info('{}Starting {}...'.format(os.linesep, sf133.full_file))
-        loadSF133(
-            sf133.full_file, file_match.group('year'), file_match.group('period'))
+    # todo: once the broker init scripts are consolidated, remove next 4 lines
+    if localSF133Dir:
+        loadAllSf133(localSF133Dir)
+    else:
+        loadAllSf133()
```

Comment:
Indeed. These lines are slated for demise, once the original validator init script gets deleted. Since I've been working with Katie on sandbox and think we'll be able to coordinate this sprint, I'll just update the PR with those deletes so there's no confusion.